dianna/methods/lime.py

```diff
@@ -1,6 +1,8 @@
 import numpy as np
+from lime.lime_image import ImageExplanation
 from lime.lime_image import LimeImageExplainer
 from lime.lime_text import LimeTextExplainer
+from numpy.typing import NDArray
```

Comment:
```suggestion
```
Not necessary. The returned type can just be `np.array`, see below.

Reply:
Ah yes, I have used this from the np.typing module as it supports hinting towards dimensions and types, but I never end up doing that. np.array is a higher level function to create an np.ndarray object which is what all arrays in numpy are defined as. So shouldn't we use `np.ndarray`?

Reply:
Yes, I think `np.ndarray` is much better. Thanks for the suggestion.

Reply:
I guess this is now consistent for now with the rest of the codebase so lets leave it as `np.array` for now but maybe this could be listed as a more general issue.